firebase_apis/test_api/src/main/java/com/google/testing/model/ApkManifest.java

```diff
@@ -71,6 +71,13 @@
   @com.google.api.client.util.Key
   private java.lang.Integer targetSdkVersion;
 
+  /**
+   * Permissions declared to be used by the application
+   * The value may be {@code null}.
+   */
+  @com.google.api.client.util.Key
+  private java.util.List<java.lang.String> usesPermission;
```

Comment:
Should we add related logic to handle the new feature? We did it that way in case of `fail-fast`.
Flank supports it even though it's not visible in the official documentation https://cloud.google.com/sdk/gcloud/reference/alpha/firebase/test/android/run

That's an open question :)

Reply:
It just updates client with a new option, we will not use it so it is safe to have it ready to deploy in codebase :D

Reply:
True, but what's the point of updating the client if we do not use new features. Leaving it as it is would be safer then.

Reply:
I don't insist on implementing it. I think it would be great to be concise. If flank supports one feature that is available in client API why not all of them? If we don't want to have all, how do we filter which to implement and which not...
That all comes to the question -- if flank should be proactive and implement new features only when requested by users only or on the contrary, adopt as fast as possible.

Reply:
I think it is good to have an always up to date client 🙂 we can track using the new features in new tickets.

Reply:
Perfect!